.github/workflows/build.yml

```diff
@@ -153,31 +150,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4']
+        php: ['8.0', '8.1', '8.2', '8.3', '8.4']
     steps:
       - uses: actions/checkout@v4
+
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: "${{ matrix.php }}"
           ini-values: "phar.readonly=0,zend.exception_ignore_args=Off"
           coverage: none
 
+      - name: Remove vimeo/psalm until it supports PHP 8.4
+        if: matrix.php == '8.4'
+        run: composer remove vimeo/psalm --no-update --dev
+
```

Comment:
This shouldn't be needed - we're not running psalm in this job?

Reply:
it is needed to remove from dev deps or `composer install` will fail